### DISCUSSION TITLE
Midi in/out in VST mode?

### DISCUSSION THREAD

**Discussion by xeroooxed** [plugdata-team/plugdata#1432] [Ideas]
Hi! I'm wondering if it is possible to implement the midi in/out in VST mode? or maybe the master app will prevent the VST to send midi?
Case scenario: 
-Using plugdata as a VST instrument in Ableton. 
-Working with processing.js for the visuals.
-Sending midi cc from plugdata (as a VST) to processing to control/sync the visuals with the music. 
I can do this from plugdata standalone, but in VST mode there is no option to send midi. 
Also, having midi in would allow to parent midi controllers to the sliders in plugdata (as a VST).
Thanks for your attention 👯‍♂️

**Comment by timothyschoen** [comment]
Ah, yeah I think this is what you mean: https://github.com/plugdata-team/plugdata/issues/1248

I should implement it :)

**Reply by xeroooxed** [reply]
It is! sorry for the duplicate topic. It would be indeed a very powerful implementation. Thanks for your attention!

**Comment by dromer** [comment]
Why can't you receive/send midi to/from ableton and into/out of plugdata?
